.github/workflows/package-build-check-publish.yml

```
@@ -100,13 +95,7 @@ jobs:
       - name: Publish and tag on NPM repository
-        if: ${{ inputs.publish-to-gar }}
         run: |
           rm -f ~/.npmrc .npmrc .yarnrc
           echo "//europe-west1-npm.pkg.dev/forto-artifacts/${{ inputs.gar-registry }}/:_authToken=\${FORTO_ARTIFACTS_TOKEN}" >> .npmrc
-          yarn publish --registry https://europe-west1-npm.pkg.dev/forto-artifacts/${{ inputs.gar-registry }}/ --tag canary
-      - if: ${{ !inputs.publish-to-gar }}
-        run: |
-          rm -f ~/.npmrc .npmrc .yarnrc
-          echo -e "always-auth=true\n//verdaccio.forto.io/:_authToken=\${VERDACCIO_FORTO_IO_TOKEN}\nregistry=https://verdaccio.forto.io" >> ~/.npmrc
-          yarn publish --registry https://verdaccio.forto.io --tag ${{ steps.tag.outputs.pre_release_version }}
+          yarn publish --registry https://europe-west1-npm.pkg.dev/forto-artifacts/${{ inputs.gar-registry }}/ --tag canary
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)